### PR TITLE
Default App retries

### PR DESCRIPTION
**Pull request by benjaminpkane**
Changes the default number of retries for all network requests to 2

### CODE REVIEW

**Review by brimoor — APPROVED** [review]
LGTM

File: app/packages/utilities/src/fetch.ts [hunk 1]
```diff
@@ -89,11 +89,11 @@ export const setFetchFunction = (
     const fetchCall = retries
       ? fetchRetry(fetch, {
           retries,
-          retryDelay,
+          0,
           retryOn: (attempt, error, response) => {
             if (
               error !== null ||
-              (response.status >= 400 && attempt < retries)
```
Comment (brimoor) [inline app/packages/utilities/src/fetch.ts:96]: Wait I'm confused; isn't this PR reducing the number circumstances in which retries are performed?

Comment (benjaminpkane) [reply app/packages/utilities/src/fetch.ts:96]: Yes. But why would we want to retry in all of those scenarios?

We retry on the event source indefinitely to check for liveliness, but we really shouldn't be retrying on other requests. An error means something isn't working...

Comment (brimoor) [reply app/packages/utilities/src/fetch.ts:96]: I was asking bc I didn't see how this PR was adding anything, but I missed that `retries` used to be 0

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**